ccm_web/server/src/auth/auth.interfaces.ts

```diff
@@ -0,0 +1,10 @@
+export interface JwtPayload {
+  username: string
+  sub: bigint
+  iat: number
+  exp: number
+}
+
+export interface MaybeCSRFError extends Error {
```

Comment:
"`Maybe`"‽ 😆

Reply:
Well, it's only possibly due to a CSRF error. The CSRF middleware throws an error/exception with a `code` property, so I wanted to type it with that property being optional.